Package.swift

```diff
@@ -19,17 +19,17 @@ import PackageDescription
 let package = Package(
     name: "TensorsFittingPerfectly",
     dependencies: [
-         .package(url: "https://github.com/tensorflow/swift", .branch("master")),
+         .package(url: "https://github.com/tensorflow/swift", .branch("main")),
          // FIXME: We need this for command-line argument parsing only.
-         .package(url: "https://github.com/apple/swift-package-manager.git", from: "0.1.0"),
+         .package(url: "https://github.com/apple/swift-tools-support-core.git", from: "0.1.12"),
```

Comment:
I still see the repo at the original URL. Why is this change needed?

Reply:
BTW if you feel like it then really an ideal fix would be to just use a different library for argument parsing.